src/main/app/src/Components/LandscapeComponent/Item/LandscapeItem.tsx

```diff
@@ -1,36 +1,46 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useState, useCallback } from 'react';
+import { get } from '../../../utils/API/APIClient';
 import './LandscapeItem.scss';
 
+import { IItem } from '../../../interfaces';
+
 interface Props {
   element: Element;
 }
 
 /**
  * Returns a choosen Landscape Item if informations are available
  * @param element Choosen SVG Element from our Landscape Component
- * TODO load assessment data
- * TODO maybe use data from landscape context
  */
 const LandscapeItem: React.FC<Props> = ({ element }) => {
-  const [html, setHtml] = useState<string>(`Not Found :(`);
+  const [item, setItem] = useState<IItem | null>();
+  const [loadItem, setLoadItem] = useState<boolean>(true);
   const [topic, setTopic] = useState<string | null>(null);
 
+  const getItem = useCallback(async () => {
+    if (loadItem && topic) {
+      setItem(await get(`/api/${topic}`));
+      setLoadItem(false);
+    }
+  }, [loadItem, topic]);
+
   useEffect(() => {
     let topic = element.getAttribute('data-identifier');
     setTopic(topic);
     if (topic !== null) {
-      console.log(topic);
-      fetch(process.env.REACT_APP_BACKEND_URL + '/api/' + topic)
-        .then((response) => {
-          return response.json();
-        })
-        .then((data) => {
-          setHtml(JSON.stringify(data));
-        });
+      getItem();
     }
-  }, [element, topic, html]);
+  }, [element, topic, getItem]);
 
-  return <div className='landscapeItemContent'>{html}</div>;
+  return (
```

Comment:
We can leave this for now. I've made a wireframe to discuss future changes.

Reply:
no we cant because it doesnt work anymore this way, we wont get html served anymore and we cant convert json to html :/

Reply:
oh or do you mean styling the component? that's something that we can leave for now yes :D the slide in from the right side was the next thing i wanted to do